sklearn/cross_decomposition/pls_.py

```diff
@@ -426,7 +426,7 @@ def fit_transform(self, X, y=None, **fit_params):
             Training vectors, where n_samples in the number of samples and
             q is the number of response variables.
 
-        copy : boolean
+        copy : boolean, default True
```

Comment:
I agree with @jnothman. I don't like "optional" in 90% of the cases but we should keep this pr on topic.